tsl/test/expected/partial_aggs.out

```diff
@@ -0,0 +1,224 @@
+$$);
+INSERT INTO foo values( 1, '2005-01-01 00:00:00+00', NULL);
+INSERT INTO foo values( 2, '2005-01-01 00:00:00+00', NULL);
+--SELECT * FROM chunk_relation_size_pretty('foo');
+create or replace  view v1(a, partialb, partialtv) as select a, _timescaledb_internal.partialize_agg( max(b) ), _timescaledb_internal.partialize_agg( min(toastval)) from foo group by a;
+create table t1 as select * from v1;
+-- 2145 min(text), 2127 max(timestamp)
+-- select regprocedureout( 2127);
+insert into t1 select * from v1;
+select a, _timescaledb_internal.finalize_agg( 'max(timestamp with time zone)', null, null, partialb, null::timestamptz ) maxb, substring(_timescaledb_internal.finalize_agg( 'min(text)', 'pg_catalog', 'default', partialtv, null::text ) from 1 for 20) mintv
+from t1 group by a order by a;
+ a |             maxb             |        mintv         
+---+------------------------------+----------------------
+ 1 | Wed Oct 19 10:23:54 2005 PDT |  I am a tall big gir
```

Comment:
we should check that the outputted string is equal to the inputted one, or at least has the same length